src/Extensions/Diagnostics.HealthChecks/Internal/HttpEndpointHealthCheck.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Fabric;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Omex.Extensions.Abstractions;
+using Microsoft.Omex.Extensions.Abstractions.Activities;
+
+namespace Microsoft.Omex.Extensions.Diagnostics.HealthChecks
+{
+	internal class HttpEndpointHealthCheck : AbstractHealthCheck
```

Comment:
>HttpEndpointHealthCheck [](start = 16, length = 23)

Would it be possible to add some logging to be able to troubleshoot if anything goes wrong with the healthcheck logic?

Reply:
Good point, added  more log calls, thank you